Subtester/Packages/manifest.json

```diff
@@ -1,5 +1,7 @@
 {
   "dependencies": {
+    "com.revenuecat.purchases-unity": "file:../../RevenueCat",
+    "com.google.external-dependency-manager": "1.2.172",
```

Comment:
@vegaro @aboedo  I missed that after I added `com.google.external-dependency-manager` as a dependency in the `RevenueCat` package, I started getting an error in the editor due to not finding this package as a dependency in Subtester. It seemed to still be able to compile fine but in order to remove that error, I removed the version of this dependency that was checked in the repo and added it here as a package registry dependency. A couple of things to note:
- During the `unitypackage` creation script, I had to remove this dependency as well, to avoid duplicate files errors.
- We are using the latest version available in openupm (1.2.172) but the latest version of the dependency is 1.2.175. I think we should try to update to the latest soon, either by submitting the new version to the existing openupm dependency (which would require updating this repo: https://github.com/xinatcg/GoogleExternalDependencyManager) or opening our own repo to contain the externaldependencymanager as a package and we have more control over it.

Reply:
> During the unitypackage creation script, I had to remove this dependency as well, to avoid duplicate files errors.
We need to package it to the unitypackage though https://github.com/googlesamples/unity-jar-resolver#plugin-redistribution , are we not packaging it now inside the unitypackage? I think we need to include it for whoever is not using the unity package manager correct?

Reply:
Isn't the EDM4U plugin available in the Unity Package Manager? Can we say our plugin depends on that one? It probably requires adding another repository, I am not sure

Reply:
Ok, this has recently changed according to this post https://forum.unity.com/threads/add-external-dependency-manager-via-package-manager.1345652/ 

I knew I had installed it using the Package Manager before haha

Reply:
Right, so basically Subtester now uses the EDM4U version from openupm (the old scoped registry stopped being updated [as you said ](https://forum.unity.com/threads/add-external-dependency-manager-via-package-manager.1345652/)).

When we export the `unitypackage` though, we remove both the `purchases-unity` and `EDM4U` package from Subtester. Then, we add a symbolic link to the purchases-unity folder and download the latest version from the [unity-jar-resolver](https://github.com/googlesamples/unity-jar-resolver) repo and import it into the Subtester/Assets folder. This is quite complicated and hacky... We could create another project that is only used for exporting the `unitypackage`, which might be simpler, but as you said, on the `unity-jar-resolver` instructions, they say we need to import the files on the export process, quite weird... 

In any case, the current script also generates a unitypackage with EDM4U included.

Reply:
Regarding updating the version of EDM4U, I am a bit concerned about using that repository... It only has 6 commits and it looks abandoned. It might not be a bad idea to host it ourselves to be honest, but I wonder if we would be breaking the license or something like that. Also, there's no guarantee that's the original plugin code, and not a modified one

Reply:
Yeah, I'm also a bit concerned about that... I think at this point we could either:

1. Remove the package dependency and just keep EDM4U as a unitypackage and ask devs to import it manually when using purchases-unity as a package. This seems to be the approach Qonversion follows (they ask devs to download it and import it manually to their projects).

2. Host it ourselves. There could be problems with the license as you said and also, there could be a conflict within openupm since I'm not sure how it would handle if I tried to publish a package with the same name as an existing package: https://github.com/openupm/openupm/blob/73af27d67d1553385b2088b293573056f27fa630/data/packages/com.google.external-dependency-manager.yml.  We probably would need to give it a different name.

Honestly, 1 is much simpler for us but would require more work for devs to use our SDK... 2 has more risks and we would also need to maintain this repo. I'm personally leaning towards 1 though... Any thoughts? @vegaro @aboedo

Reply:
is there any form of post-install trigger we can use to run a script, to do this on behalf of the developer? UPM does have a scripting API

Reply:
Oh that's an interesting idea! I took a quick look at it and I think it could potentially work but will need to investigate a bit more. I believe we would need to have a script that does something like:
- If EDM4U not found in project (figuring this out can be difficult if the dev has it in a non-conventional location...):
   - Download latest `unitypackage` from EDM4U repo
   - Import `unitypackage` into project (we have to see if we can do this from these scripts)
- If EDM4U found in project, do nothing.

There is some complexity there but if it works, it could make the installation easier using UPM. That said, note that EDM4U only needs to be installed once, but `purchases-unity` can receive upgrades and it shouldn't need updating EDM4U, so I'm wondering if it's worth the extra complexity...

Reply:
well, we can start without it and then if people are having a rough time with EDM4U then figure out workarounds 👍